addons/mail/static/src/messaging/component/notification_group/notification_group.js

```diff
@@ -0,0 +1,125 @@
+odoo.define('mail.messaging.component.NotificationGroup', function (require) {
+'use strict';
+
+const useStore = require('mail.messaging.component_hook.useStore');
+
+const { Component } = owl;
+
+class NotificationGroup extends Component {
+
+    /**
+     * @override
+     */
+    constructor(...args) {
+        super(...args);
+        useStore(props => {
+            const group = this.env.entities.NotificationGroup.get(props.notificationGroupLocalId).__state;
```

Comment:
Do we really want to add those conditions when they are not manifesting? Isn't the goal to get rid of them?

Reply:
maybe you don't have this problem on your branch but it sucks if other dev branches have this issue, waste time fixing this issue (for xdu and jgi, it may take some time before they get this is where the issue comes from), while you could have just guarded it in this PR before merge.

Reply:
I disagree but this is not important enough to care. I'll just add it to be sage.